docs/mobile-apps/features/mobile-app-diagnostics/app-crash-logs.md

```diff
@@ -21,15 +20,42 @@ Our tool is capable of detecting and recording fatal errors (crashes) that occur
 
 ## Viewing the App Crash Logs for Real Devices​
 
-The App Crash logs feature is automatically enabled when the instrumentation feature is enabled. Whenever a crash is detected, our system will display an App Crashed label for the corresponding test result, regardless of the test's status.
+The App Crash Logs feature is automatically activated when the [Instrumentation feature](/mobile-apps/features/mobile-app-diagnostics/interactions/#using-interactions-for-real-devices) is enabled. Upon detecting a crash, our system will exhibit an `App Crashed` label for the corresponding test result, regardless of the test's status.
 
 <br/><img src={useBaseUrl('img/mobile-apps/app-crash-scr1.png')} alt="Mobile app settings navigation" width="900"/>
 
-When you navigate to the [Live Test Results](/mobile-apps/live-testing/live-mobile-app-testing/) or [Automated Test Results](/mobile-apps/automated-testing/) page, you can view all the collected crashes in a centralized location. The collected crashes are presented in the context of their respective test cases, videos, and interactions. This centralized view is designed to enable efficient collaboration within your team and improve time to resolution for any issues that may arise.
+When you navigate to the [Live Test Results](/mobile-apps/live-testing/live-mobile-app-testing/) or [Automated Test Results](/mobile-apps/automated-testing/) page, you can view the collected crashes in the Crashes tab. The collected crashes are presented in the context of their respective test cases, videos, and interactions. This centralized view is designed to enable efficient collaboration within your team and improve time to resolution for any issues that may arise.
+
+### Viewing the App Crash in the Crashes Tab
+
+<img src={useBaseUrl('img/mobile-apps/app-crash-scr2.png')} alt="Mobile app settings navigation" width="800"/>
+
+The `crash.log` file provides information on the stack trace, methods, classes, and lines of code that were being executed before the fatal error occurred. This information facilitates the identification of the root cause of the crash, enabling your team to address the underlying issues in a timely and effective manner.
+
+In the Crashes tab, you will also find a **View Crash in Error Reporting** link, which directs you to [Error Reporting](/error-reporting/getting-started/) for a detailed view of the crash logs.
+
+<img src={useBaseUrl('/img/mobile-apps/rdc-er-integration/crashes-link.png')} alt="view crash in error reporting link in Crashes tab" width="700" />
+
+### Viewing the App Crashes in Error Reporting
+
+
+While the Crashes Tab contains the stack trace, which can be useful for initial debugging of the root cause, more comprehensive insights are available within Error Reporting. Moreover, Error Reporting is the sole location to access callstack details for native crashes.
```

Comment:
we should add something about link from the sidebar to the backtrace instance, below Mobile Beta Testing

We can address this functionality in an other PR

<img width="211" alt="image" src="https://github.com/saucelabs/sauce-docs/assets/68342451/b0222360-e586-40dc-b221-641adace94eb">